solidity/test/token_stake/TestTokenStakingEscrow.js

```diff
@@ -273,7 +275,7 @@ describe('TokenStakingEscrow', () => {
 
     it('can not be called by third-party', async () => {
       await expectRevert(
-        escrow.withdraw(operator, {from: deployer}),
```

Comment:
Maybe we should also leave the cases to test if `deployer` is not able to call the function?